src/documentation/templates/component-inputs.html

```diff
@@ -49,17 +49,57 @@ <h3 class="m-t-48">
                       </div>
                     </label>
                   </li>
-                  <li class="field-item col-sm-4 m-b-12 awa-form">
-                    <!-- <label for="lastname" aria-label="text-input">
-                      Name
-                      <input
-                        type="text"
-                        id="lastname"
-                        name="lastname"
-                        placeholder="Last name"
-                        aria-invalid="false"
-                      />
-                    </label> -->
+                  <li class="field-item col-sm-8 m-b-12">
+                    <label
+                      for="password"
+                      class="labelpassword"
+                      data-required="true"
+                    >
+                      Contraseña:
+                      <div class="dp-wrap-eyed">
+                        <button
+                          id="see"
+                          aria-label="see"
+                          class="show-hide icon-hide ms-icon"
+                          tabindex="-1"
+                        ></button>
+                        <input
+                          type="password"
+                          id="password"
+                          name="password"
+                          placeholder="Your Password"
+                          aria-required="true"
+                          aria-invalid="false"
+                          aria-placeholder="Your Password"
+                        />
+                      </div>
+                      <div class="wrapper-password">
+                        <p class="password-message">
+                          <span class="dp-message--success"
+                            >8 caracteres como mínimo</span
+                          >
+                          <span class="dp-message--success">Una letra</span>
+                          <span class="dp-message--success">Un número</span>
+                        </p>
+                        <p class="password-message">
+                          <span class="dp-message--denied">
+                            8 caracteres como mínimo</span
+                          ><span class="dp-message--denied"> Una letra</span
```

Comment:
Relacionado con [este comentario](https://github.com/FromDoppler/doppler-style-guide/pull/235/files#r1140345319), lo parto en _suggestions_ más pequeñas para que sea más claro.

```suggestion
                          ><span class="dp-message--denied">Una letra</span
```